tasks/esi_node_network_attach_task.yaml

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 2,
+    "title": "A cropped version of a bigger task.",
+    "description": "Auto-generated task from a single workload",
+    "subtasks": [
+        {
+            "title": "ScenarioPlugin.esi_node_network_attach",
+            "description": "",
+            "scenario": {
+                "ScenarioPlugin.esi_node_network_attach": {
+                    "node": "dell-2",
+                    "network": "storage",
```

Comment:
These should not be committed hardcoded. I'd change this file name to esi_node_network_attach_task.yaml.sample, and update the README to specifically mention this file and what might need to be changed.

Reply:
I have found a way to give node and network from the terminal. So no need to hard code.